core/src/mindustry/ui/fragments/MenuFragment.java

```diff
@@ -28,6 +28,30 @@ public class MenuFragment{
     private Button currentMenu;
     private MenuRenderer renderer;
     private Seq<MenuButton> customButtons = new Seq<>();
+    public MenuButton[] desktopButtons;
```

Comment:
Why is this an array and not a Seq?

Also, should probably be nullable. Or just an empty array/seq on mobile.